### PR TITLE
#161966874 User Profiles

### PR DESCRIPTION
**Pull request by leonardnjura**
#### What does this PR do?
- Integrates route to access user profile
- Allows user to edit profile

#### How should this be manually tested?
`$ git clone -b ft-user-profiles-161966874 https://github.com/andela/ah-infinity-stones-frontend`
`$ cd ah-infinity-stones-frontend`
> Install node modules specified in `package.json`
`$ npm install`
`$ npm start`

> Navigate to `~/login` page
> Then Navigate to `~/profile` page
> If you are authenticated, now you are ready to access and edit your profile

#### What are the relevant pivotal tracker stories?
[#161966874](https://www.pivotaltracker.com/story/show/161966874)

#### Screenshots
screen shot 2019-01-25 at 01 58 32
screen shot 2019-01-25 at 02 04 53
screen shot 2019-01-25 at 02 04 01

### CODE REVIEW

**Review by kamyaD — APPROVED** [review]

**Review by codeplus254 — APPROVED** [review]
Great work!!!

File: src/components/Profile/containers/ProfileForm.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  getProfile: PropTypes.func.isRequired,
+  updateProfile: PropTypes.func.isRequired,
+  firstname: PropTypes.string.isRequired,
+  lastname: PropTypes.string.isRequired,
+  birthday: PropTypes.string.isRequired,
+  image: PropTypes.string.isRequired,
+  bio: PropTypes.string.isRequired,
+};
+
+const mapDispatchToProps = state => ({
```
Comment (johnmutuma5) [inline src/components/Profile/containers/ProfileForm.js:176]: @leonardnjura, the name here is misleading, do you mean `mapStateToProps`?

File: src/redux/sagas/profileSagas.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { put, takeLatest } from 'redux-saga/effects';
```
Comment (johnmutuma5) [inline src/redux/sagas/profileSagas.js:1]: Nice job on the sagas @leonardnjura

**Review by johnmutuma5 — APPROVED** [review]
Leo, Nice work. Please see some comments and address the naming issue mentioned in the next PR.

File: src/components/Profile/containers/ProfileForm.js (hunk 1 above)
Comment (leonardnjura) [reply src/components/Profile/containers/ProfileForm.js:176]: @johnmutuma5 well noted, will fix it

**Pull request merged**